BUILD.bazel

```diff
@@ -233,6 +234,19 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "spirv_tools_lint",
+    srcs = glob(["source/lint/*.cpp", "source/lint/*.hpp"]),
```

Comment:
There are no `hpp` files in `source/lint`.  Are you expecting any?  In any case, headers under `source/` have `.h` extension.

Reply:
I added this so that in the near future when we add headers, we won't have to modify the build files. I can remove if you want.

Reply:
Yeah, thanks.  It's slightly cleaner if they only get added when needed.